packages/fast-components/src/styles/patterns/button.ts

```diff
@@ -0,0 +1,235 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../display";
+import { focusVisible } from "../focus";
+
+export const BaseButtonStyles = css`
+    ${display("inline-block")} :host {
+        font-family: var(--body-font);
+        outline: none;
```

Comment:
You probably need to rebase - the design-system implementation went in so you'll need to declare which recipes this component depends on.